controllers/account/controllers/account_controller.go

```diff
@@ -151,6 +160,38 @@ func (r *AccountReconciler) syncAccount(ctx context.Context, name, accountNamesp
 	if err := r.syncRoleAndRoleBinding(ctx, name, userNamespace); err != nil {
 		return nil, fmt.Errorf("sync role and rolebinding failed: %v", err)
 	}
+
+	// add account balance when account is new user
+	stringAmount := os.Getenv(NEWACCOUNTAMOUNTENV)
+	if key, ok := account.Annotations[AccountAnnotationNewAccount]; (ok && key == "false") || (stringAmount == "") {
```

Comment:
done,this case will not ，because before i will set `account.Annotations[AccountAnnotationNewAccount]=true`
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/87080562/222473787-2f695522-10cf-4515-bdc9-7fc0753f1439.png">